client/src/components/main/MainContainer.js

```diff
@@ -17,7 +17,7 @@ export default () => {
   return (
     <React.Fragment>
       <TopBar />
-      <Switch location={ background || location }>
+      <Switch location={background || location}>
```

Comment:
Its recommended by pretty much any JS style guide to include spaces inside of curly braces, [The airbnb stye guide for example](https://github.com/airbnb/javascript#whitespace--in-braces)

Reply:
This was an issue with the `eslint` config, I'm not sure why only you are getting this behavior but the new config should fix this.